presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -163,7 +171,9 @@ public void testCurrentTime()
     @Test
     public void testLocalTimestamp()
     {
-        functionAssertions.assertFunction("localtimestamp", TimestampType.TIMESTAMP, toTimestamp(session.getStartTime()));
+        ISOChronology localChronology = getChronology(session.getTimeZoneKey());
+        long utcMillisOfLocalTime = localChronology.getZone().convertUTCToLocal(session.getStartTime());
```

Comment:
These are clockMillis (representation of wall time) ready to be saved in TIMESTAMP W/O TZ. Name localMillis is not accurate and suggest bug in code as it's quickly passed directly to millisUtc field inside SqlTimestamp.
I think this name is not perfect, but still like it better then the one proposed by you and I'm not sure about introducing clockMillis concept here.